code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -0,0 +1,703 @@
+    * __onboardArtifact__  Submits an artifact to operator platform
+    * __removeArtifact__  Removes an artifact from operator platform
+    
+    __MEC Platform information__
+    *  __retrieveEdgeNodes__ List of the operator’s MECs and their status, ordering the results by location and filtering by status (active/inactive/unknown)
+       
+    # Further info and support
+  
+    (FAQs will be added in a later version of the documentation)
+
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: sp-edc@lists.camaraproject.org
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+externalDocs:
+  description: Product documentation at Camara
+  url: https://github.com/camaraproject/
+
+servers:
+  - url: "{apiRoot}/{basePath}"
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+      basePath:
+        default: edge-cloud/v1
+        description: Base path for the Edge Cloud API
+
+security:
+  - oAuth2ClientCredentials: 
+    - nbi-mgmt
+
+tags:
+  - name: Applications
+    description: Manage of Applications
+  - name: Application Artifacts 
+    description: Manage of Artifacts
+  - name: Mobile Edge Cloud Platform
+    description: Mobile Edge Nodes Available
+
+paths:
+  /app:
+    post: 
+      tags:
+        - Applications
+      summary: Deploy an application 
+      description: Ask the operator to provision an application to one or several Edge Application Servers taking into account resources (e.g. vCPU, Memory, network interfaces, storage, GPU) 
+      operationId: appProvisioning
+      requestBody:
+        description: Details about application and zones where application instance should be instantiated
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AppProvisioning'
+        required: true      
+      responses:
+        '201':
+          description: Application provisioned
+          content: 
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref:  '#/components/schemas/DeployedApp'
+                minItems: 1                 
+        '400':
+          $ref: '#/components/responses/400'
+        '401':
+          $ref: '#/components/responses/401'
+        '403':
+          $ref: '#/components/responses/403'
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorInfo'
+              example:
+                status: 409
+                code: CONFLICT
+                message: "Application already deployed"
+        '500':
+          $ref: '#/components/responses/500'
+        '501':
+          $ref: '#/components/responses/501'
+        '503':
+          $ref: '#/components/responses/503'
+
+  /app/{appId}:
+    delete:
```

Comment:
As discussed in the meeting there should be a GET method also to retrieve the termination status of the app instance.

Reply:
We finally include this GET methods for app and appinstance in the last v0.2.0 
It has been included a status parameter for app instance including the following possible states:
          - ready
          - instantiating
          - failed
          - terminating
          - unknown